redis-persistence/src/main/java/com/netflix/conductor/redis/config/DynomiteClusterConfiguration.java

```diff
@@ -35,14 +36,26 @@ protected JedisCommands createJedisCommands(
         ConnectionPoolConfigurationImpl connectionPoolConfiguration =
                 new ConnectionPoolConfigurationImpl(properties.getClusterName())
                         .withTokenSupplier(tokenMapSupplier)
-                        .setLocalRack(properties.getAvailabilityZone())
                         .setLocalDataCenter(properties.getDataCenterRegion())
                         .setSocketTimeout(0)
                         .setConnectTimeout(0)
                         .setMaxConnsPerHost(properties.getMaxConnectionsPerHost())
                         .setMaxTimeoutWhenExhausted(
                                 (int) properties.getMaxTimeoutWhenExhausted().toMillis())
-                        .setRetryPolicyFactory(properties.getConnectionRetryPolicy());
+                        .setRetryPolicyFactory(properties.getConnectionRetryPolicy())
+                        /*
+                         * default LBStrategy is TokenAware.
+                         * Using RoundRobin so that requests get distributed to other nodes in dynomite cluster.
+                         *
+                         * In our topology we have only one node per rack and availabilityZone(localRack) set to a rack.
+                         * Dyno client will try to Roundrobin within the localRack, which results in all requests
+                         * going to that one node in that specific rack.
+                         *
+                         * Given the above topology, for roundrobin LB strategy to work we need to set availabilityZone to null.
+                         * So that there is no localRack affinity and requests get distributed.
+                         */
+                        .setLoadBalancingStrategy(LoadBalancingStrategy.RoundRobin)
+                        .setLocalRack(null);
```

Comment:
Instead of making this change in code, isn't it better to remove this setting from the config 
https://bitbucket-eng-sjc1.cisco.com/bitbucket/projects/AN/repos/galaxy/browse/docker/config/dynomite-cluster.config.properties#78